Nasal/c172p.nas

```diff
@@ -43,14 +43,15 @@ var autostart = func (msg=1) {
     setprop("/consumables/fuel/tank[0]/water-contamination", 0.0);
     setprop("/consumables/fuel/tank[1]/water-contamination", 0.0);        
 
-    # Oil level
-    var oil_enabled = getprop("/engines/active-engine/oil_consumption_allowed");
-    var oil_level   = getprop("/engines/active-engine/oil-level");
-
-    if (oil_enabled and oil_level < 6.0)
```

Comment:
Could you add `if (oil_enabled and oil_level < 6.0)` around the new if-else? You can keep the if-else, but add this old condition around it. It was added to avoid completely resetting the oil level when someone just has a (for once) lazy moment.